antenna-documentation/src/site/markdown/sw360-data-model.md.vm

```diff
@@ -40,31 +41,36 @@ In the following section there is an outline what data of SW360 ${docNameCap} ma
-  | Declared License | The licenses declared by the release in a SPDX license expression that displays the relationship of the licenses as one String. |
-  | Observed License | The licenses observed during the ${docName} run in the SPDX license expression that displays the relationship of the licenses as one String. |
-  | Release Tag Url | This is a link to the OSS repos tag of the release used. |
-  | Software Heritage ID | A release ID in software heritage in the format "swh:1:rel:*" |
-  | Hashes | A set of hashes associated with the release. |
-  | Clearing State | Data Approval State |
-  | Change Status | OSS Usage |
-  | Copyrights | Copyrights of the release, given in a String. Individual copyrights are separated by line. |
- 
\ No newline at end of file
+  | Component ID | ID of the component this release belongs to. E.g. The ID of the Junit component. |
+  | Name | Name of the release, which should equal the component name store in the `Name` field. |
+  | Version | Version of the release stored in the `Version` field. |
+  | Download Url | Local link to the source archive stored in the `Download URL` field. I.e., the link to the local storage where the sources of the release are stored. |
+  | Main License IDs | IDs of the final licenses in the `Main License Ids` field. The IDs have been extracted from the license information below and link to licenses in the License database. |
+  | Purls | These are the corresponding release coordinates that are saved under an external ID as purls. It is a list of key-value pairs of Strings, with the purl type as key, e.g., `maven`, and the purl as value. There can be multiple coordinates. |
+  | Overwritten License | The manually defined license of the release in a SPDX license expression represented as String in the additional data with `overwritten_license` as key. |
+  | Declared License | The declared licenses of the release in a SPDX license expression represented as String in the additional data with `declared_license` as key. |
+  | Observed License | The obsserved licenses of the release in a SPDX license expression  represented as String in the additional data with `observed_license` as key. |
+  | Release Tag Url | This is a link to the OSS repos tag of the release used stored as external id with key `orig_repo`. |
+  | Software Heritage ID | A release ID in software heritage in the format "swh:1:rel:*", stored as external id with key `swh`. |
+  | Copyrights | Copyrights of the release, given in a String. Individual copyrights are separated by line. Stored in the additional data section with key `copyrights` |
+  | Change Status | An enumeration with values `changed` or `as-is` stored as additional data with key `change_status`. A changed component release should be stored as own release with an appendix to the original version, e.g. `1.2.3_modified_xyz` |
```

Comment:
How would we enforce that a changed component release is stored with a new name?
This isn't implemented or mentioned anywhere else-

Reply:
It is not a new name, but a different version, I would keep this in the documentation but would hint, that it is not implemented yet, only as-is components can be handled right now